linkerd/proxy/client-policy/src/lib.rs

```diff
@@ -175,7 +175,7 @@ impl ClientPolicy {
                 opaque: opaq::Opaque {
                     // TODO(eliza): eventually, can we configure the opaque
                     // policy to fail conns?
-                    policy: None,
+                    routes: std::sync::Arc::new([]),
                 },
```

Comment:
Similarly, this TODO is addressed, and we can use a static routes. This does suggest that we may want to follow up with a filter that fails connections with an error, as we do for HTTP. This may also serve as a mechanism for propagating denials.